WireDomain/Tests/WireDomainTests/Helpers/UUID+Mock.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireAPI
+
+extension UUID {
```

Comment:
Perhaps we should move this (and the ones in WireAPI) to a test utilities target.

Reply:
I moved it to WireFoundation > WireTesting > Utilities in https://github.com/wireapp/wire-ios/pull/2019/commits/de1f3cf2daf03969a031a3d5fc73a19402d26eb4